src/Npgsql.OpenTelemetry/README.md

```diff
@@ -20,3 +20,42 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Once this is done, you should start seeing Npgsql trace data appearing in your application's console. At this point, you can look into exporting your trace data to a more useful destination: systems such as [Zipkin](https://zipkin.io/) or [Jaeger](https://www.jaegertracing.io/) can efficiently collect and store your data, and provide user interfaces for querying and exploring it.
 
 For more information, [visit the diagnostics documentation page](https://www.npgsql.org/doc/diagnostics.html).
+
+### Enrich
```

Comment:
This is great documentation, but I'm not sure we should get into all this detail in the NuGet README; [this gets shown on nuget.org](https://www.nuget.org/packages/Npgsql.OpenTelemetry), and it seems like it should be a short summary (hence the link to the docs site above).

I've created a [7.0 branch in our docs repo](https://github.com/npgsql/doc/tree/7.0), can you please submit these docs as a PR against it (with a note that it's being introduced for 7.0)? I'll merge this a bit later once previews are out etc.

Reply:
https://github.com/npgsql/doc/pull/177